simulator/README.md

```diff
@@ -83,10 +87,44 @@ Note:
 
 1. If you want to send a boolean in Event Details, please use non capital letters, for instance `true` or `false`.
 
+### Send Telemetry to TCP Client
+
+If you want to send telemetry to TCP Client, do:
+
+- Go to tab "Send Telemetry".
+- Write a name in Telemetry Id control.
+- Write details in Telemetry Details control.
+- Click "Send one instance Telemetry" to send the only one instance.
+- Read the telemetry in TCP Client.
+
+The structure of this telemetry is, for instance:
+
+| Telemetry Id |   Telemetry Details    |
+|:------------:|:----------------------:|
+|   elevation  | "compName" : "MTMount" |
```

Comment:
In this case, maybe:
|   elevation  | "measuredInDeg" : 30 |

This is because this telemetry is published from the LabVIEW component. Therefore, you should not see the field of "compName", which happens in the subscription part.

Reply:
Good to know. Thanks!